src/connector.js

```diff
@@ -622,6 +622,20 @@ function initEngine() {
 			let site = urlParams.site.toLowerCase().replace( '*', '' );
 			aqString += ' @canadagazettesite==' + site;
 		}
+		
+		if ( urlParams.startdate && urlParams.enddate ) {
+			aqString += ' @date = "' + urlParams.startdate.replaceAll('-','/') + '..' + urlParams.enddate.replaceAll('-','/') + '"';
+		}
```

Comment:
@ipaksc Can you also include the cases where there would only be a startdate provided, and when there is only an enddate provided?

Reply:
I am not sure if this can be done as we have to provide both dates. Example  @date="2020/02/02..now/today" doesn't work for me. We can get current date with Javacript, let me know if you are fine with it

Reply:
@GormFrank How to I fix this "Aborted due to warnings.
Error: Process completed with exit code 3"?

Reply:
Done